src/components/fullscreenManager.js

```diff
@@ -0,0 +1,103 @@
+define(['events', 'dom', 'apphost', 'browser'], function (events, dom, appHost, browser) {
```

Comment:
why is this file here?

Reply:
conflicts are not resolved

Reply:
I need to rebase and re-run the fix for it to be current with master.